src/utils/config-validators/validPrehidingSelector.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import isNil from "../isNil";
+import isNonEmptyString from "../isNonEmptyString";
+
+export default (config, key, currentValue) => {
+  if (isNil(currentValue)) {
+    return "";
+  }
+
+  if (isNonEmptyString(currentValue)) {
+    return "";
+  }
+
+  return `Invalid value for ${key}: ${currentValue}`;
+};
```

Comment:
Can/Should this all be combined?

```
return isNil(currentValue) || isNonEmptyString(currentValue) ? "" : `Invalid value for ${key}: ${currentValue}`;
```

Reply:
Would it make sense to rename this validator to be more generic? It could be used for any config option that should either be nil or a non-empty string.

Reply:
@Aaronius do you think something like `validEitherNilOrNonEmpty`?

Reply:
@Aaronius regarding combining, for me it easy to read a couple of `ifs` vs ternary operator.

Reply:
If I had my druthers, I would make the usage be like this: `string().minLength(1)`

The `string()` part would validate that it's a string and the `minLength()` part would validate that it meets the minimum length (in your case, you would pass `1`).